### PR TITLE
Few fixes and cleanups

### PR DESCRIPTION
**Pull request by xclaesse**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -157,42 +148,35 @@ _files = [
...
-    'expat',
-    _files,
-    include_directories : include_directories('lib'),
-    dependencies : dep_libbsd,
-    version : '1.6.11',
-    soversion : host_machine.system() != 'windows' ? '1' : '',
-    install : true,
-    gnu_symbol_visibility : get_option('build_tests') ? 'default' : 'hidden',
-  )
+  extra_kwargs += { 'vs_module_defs' : 'lib/libexpat.def' }
```
Comment (dcbaker) [inline meson.build]: there aren't spaces around braces anywhere else in the file.

**Review by dcbaker — APPROVED** [review]
This all looks really nice, I have one tiny style nit, that's it.

File: meson.build (hunk 1 above)
Comment (xclaesse) [reply meson.build]: Ok, fixed.

**Comment by jpakkane** [comment]
I merged this by hand, meaning it did not do the release properly. Since I'm not sure how to do that properly, so rather than me breaking things even more, could @legeana look at doing it fight.

FTR the correct way of doing this can be found [on this documentation page](https://mesonbuild.com/Wrap-review-guidelines.html). It's not very discoverable, though, we should probably add a link to it from the other wrapdb doc pages.

**Comment by nirbheek** [comment]
@jpakkane this is what you're supposed to do in that case: https://github.com/mesonbuild/libopenjp2/pull/2#issuecomment-704151783

**Comment by nirbheek** [comment]
Hm, I tried that but it didn't work:

```
$ git checkout 2.2.9
Branch '2.2.9' set up to track remote branch '2.2.9' from 'origin' by rebasing.
Switched to a new branch '2.2.9'
$ mesonwrap publish expat 2.2.9 
Traceback (most recent call last):
  File "/home/nirbheek/.local/lib/python3.8/site-packages/mesonwrap/tools/publisher.py", line 41, in _import_wrap
    rel = ghrepo.get_release(tagname)
  File "/home/nirbheek/.local/lib/python3.8/site-packages/github/Repository.py", line 2887, in get_release
    headers, data = self._requester.requestJsonAndCheck(
  File "/home/nirbheek/.local/lib/python3.8/site-packages/github/Requester.py", line 317, in requestJsonAndCheck
    return self.__check(
  File "/home/nirbheek/.local/lib/python3.8/site-packages/github/Requester.py", line 342, in __check
    raise self.__createException(status, responseHeaders, output)
github.GithubException.UnknownObjectException: 404 {"message": "Not Found", "documentation_url": "https://docs.github.com/rest/reference/repos#get-a-release-by-tag-name"}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nirbheek/.local/bin/mesonwrap", line 11, in <module>
    load_entry_point('mesonwrap==0.2.0', 'console_scripts', 'mesonwrap')()
  File "/home/nirbheek/.local/lib/python3.8/site-packages/mesonwrap/cli.py", line 28, in __init__
    getattr(self, self.CMD_PREFIX + command)()
  File "/home/nirbheek/.local/lib/python3.8/site-packages/mesonwrap/cli.py", line 64, in command_publish
    publisher.main(*self.args())
  File "/home/nirbheek/.local/lib/python3.8/site-packages/mesonwrap/tools/publisher.py", line 109, in main
    publish(args.organization, args.name, args.branch)
  File "/home/nirbheek/.local/lib/python3.8/site-packages/mesonwrap/tools/publisher.py", line 97, in publish
    Publisher.publish(organization, project, branch)
  File "/home/nirbheek/.local/lib/python3.8/site-packages/mesonwrap/tools/publisher.py", line 93, in publish
    cls._import_wrap(tmp, organization, wrap)
  File "/home/nirbheek/.local/lib/python3.8/site-packages/retrying.py", line 49, in wrapped_f
    return Retrying(*dargs, **dkw).call(f, *args, **kw)
  File "/home/nirbheek/.local/lib/python3.8/site-packages/retrying.py", line 212, in call
    raise attempt.get()
  File "/home/nirbheek/.local/lib/python3.8/site-packages/retrying.py", line 247, in get
    six.reraise(self.value[0], self.value[1], self.value[2])
  File "/usr/lib/python3.8/site-packages/six.py", line 703, in reraise
    raise value
  File "/home/nirbheek/.local/lib/python3.8/site-packages/retrying.py", line 200, in call
    attempt = Attempt(fn(*args, **kwargs), attempt_number, False)
  File "/home/nirbheek/.local/lib/python3.8/site-packages/mesonwrap/tools/publisher.py", line 44, in _import_wrap
    tag = ghrepo.create_git_tag(tag=tagname, message=tagname,
  File "/home/nirbheek/.local/lib/python3.8/site-packages/github/Repository.py", line 1060, in create_git_tag
    headers, data = self._requester.requestJsonAndCheck(
  File "/home/nirbheek/.local/lib/python3.8/site-packages/github/Requester.py", line 317, in requestJsonAndCheck
    return self.__check(
  File "/home/nirbheek/.local/lib/python3.8/site-packages/github/Requester.py", line 342, in __check
    raise self.__createException(status, responseHeaders, output)
github.GithubException.UnknownObjectException: 404 {"message": "Not Found", "documentation_url": "https://docs.github.com/rest/reference/git#create-a-tag-object"}
```

**Comment by jpakkane** [comment]
It's probably because Github now requires personal access tokens to use those endpoints. Debugging...

**Comment by jpakkane** [comment]
Fixed now and in wrapdb.

**Pull request merged**